### DISCUSSION TITLE
Automatic rejections when applying for student pack

### DISCUSSION THREAD

**Discussion by lambergmiki** [community/community#138561] [GitHub Education]
### Hi everyone,

Hi, I've submitted several applications by now, and I seem to be stuck in an automatic rejection loop. I am a distance learning student and the certificates I provide clearly state that it is distance learning, however once applied I instantly receive a rejection letter adressing the fact that there isn't a campus location in my country (I study from abroad). It also questions my GitHub profile, which is completely correct with my first and last name as stated on the certificates from the school.
It also says the schools name isn't displayed on the documents, which it is TWICE!

Customer service isn't really helpful either, so I'm losing motivation to keep adressing this issue. My classmates seem to get accepted without issues.

Best regards
Miki

**Comment by MisakaRehana** [comment]
Same issue here... I was even approved two year ago, I don't know why GitHub hate us..
The school name is VERY CLEARLY in the picture, and it still automatically rejected.

**Reply by lambergmiki** [reply]
Yep, @Misaka12456. A classmate told me he sent in the same certificate, but with a snippet of the school's name zoomed in. This worked for him, so I guess I'll try that. My latest application seems to be reviewed now, though, at least it hasn't been automatically denied yet - so after that if it should fail again.
